layouts/pricing/list.html

```diff
@@ -122,6 +122,14 @@ <h3>Contact us</h3>
         <div>
             4: managed cloud storage, limit of 5 GB per seat
         </div>
+        <div>
+            If you are a student or an educator, you can avail 6 months educational access to LocalStack through 
+            our <a href="/educational-license" target="_blank" class="text-white"><u>Educational License program</u></a>.
```

Comment:
Not sure about the `target="_blank"` - personally not a big fan of it, as users should be able to decide for themselves whether to open a link in the same tab or a new tab, especially if we're linking to an internal page on our own website.
```suggestion
            our <a href="/educational-license" class="text-white"><u>Educational License program</u></a>.
```